charts/image-resizer/Chart.yaml

```diff
@@ -21,3 +21,8 @@ version: 0.2.3
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 appVersion: 1.0.0
+
```

Comment:
you need to bump the chart version up, as you made changes to the chart and values

Reply:
I didn't make changes to this chart, I only added more information to the Chart, but I can give it a patch version bump.